sensor_msgs/RegionOfInterest roi
---
Detection[] detections
sensor_msgs/Image image